modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCaseTypesAdministrator.testcase

```diff
@@ -132,4 +132,30 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-166555 - Verify if as an Administrator I should not be able to add a duplicate cases types"
+	@priority = "5"
+	test AdminCanNotAddDuplicateCasesTypes {
+		task ("Go to Case Types on Manage Sidebar and change pagination") {
+			ApplicationsMenu.gotoSite(site = "Testray 2");
+
+			TestrayLayout.clickManageOnSidebar();
+
+			TestrayManage.selectOptionsManage(itemValue = "Case Types");
+
+			Testray.changePaginationInModal(
+				numberItems = "20",
+				valuePagination = "150");
+		}
+
+		task ("Create Case Type duplicate and assert the error message should appear") {
+			TestrayManage.createNewCaseTypes(caseName = "Case_Type");
+
+			TestrayManage.createNewCaseTypes(caseName = "Case_Type");
+
+			Alert.viewErrorMessage(errorMessage = "The Case Type name already exists");
+
+			TestrayManage.closeModalInOptions();
```

Comment:
Hi @joelrocha68, this step is not necessary, since the purpose of your test is to validate the error message